packages/wd-base/src/_typography.scss

```diff
@@ -0,0 +1,123 @@
+/* ==========================================================================
+   Typography
+   ========================================================================== */
+
+/* Font Faces
+   ========================================================================== */
+$font-primary: "Circular", sans-serif !default;
+$font-secondary: "Open Sans", sans-serif !default;
+$font-mono: "Roboto Mono", sans-serif !default;
+
+
+// color
+$color-black: #0e141a;
+
+// font weight
+$display: 900;
+$emphasis: 500;
+$body: normal;
+
+// font family & color helper mixin
+@mixin font-primary($color-variable) {
+  font-family: $font-primary;
+  color: $color-variable;
+}
+
+
+/* Font Mixins
+   ========================================================================== */
+@mixin wd-typography-large($weight-variable) {
+  font-weight: $weight-variable;
+  font-size: 85px;
+  line-height: 96px;
+
+  @include font-primary($color-black);
```

Comment:
we should just handle typography stuff here in this file. let's move the colors to another place.